### PR TITLE
Final Version of 6th Microverse Project

### PR DESCRIPTION
**Pull request by nrqrmz**
### Last version of the 6th project.

- The focus of this project is responsiveness.

- Three stylesheets were created to accomplish this. One for 300px resolution, other for 768pc resolution and the last one for 1024px resolution.

### CODE REVIEW

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### Media queries

#### You have to use media queries, instead of each CSS file for one approach.
- [x] Remove all your CSS files and keep it just one file, in that add `media queries` rules. Check something about media queries [in here](https://www.w3schools.com/css/css3_mediaqueries.asp) 
    -  You have to add these breakpoints

* medium: min-width around 768px
* large: min-width 1024px

**Note:**  Please start working first with this task, once you finish moving to the other ones

### Navigation bar

**Medium screen**

- [x] Fix links on the main navbar
- [x] Links you have on the right shouldn't be in there

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

### Top three article section

**Large screen**

- [x] It should contain TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/))

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

**Medium screen**

- [x] Fix your text info in two articles set up on the right

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

**Small screen**

- [x] Each article should have a greater height
- [x] Also your text should be a small one

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

**Categories (Apps, Gear, Tech, etc) section**

Large, medium and small screens

- [x] Articles should be a square 
- [x] At the bottom of each article should have a  dark shadow

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

### Footer

**Medium screen** 

- [x] Should be a small footer

image

**Small screen**

- [x] Fix all our footer 

| ACTUAL WEB PAGE | YOURS |
| :-------------: | :---: |
| image | image |

### Summary

- Please work on your media queries first then work on suggested changes

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by nrqrmz** [comment]
Fix all changes required

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1406 @@
...
+      <div class="trigrid">
+        <div class="trigrid-article1">
+          <img src="assets/tnw.svg" class="tglogo" alt="">
+          <div class="tga-date">
+            <p class="tga-date-p">
+            </p>
+          </div>
+          <div class="tga-info">
+            <a href="#" class="tga-category bold700 uppercase no-underline">
+              insights
```
Comment (expjazz) [inline index.html:192]: - [x] This section is being covered by the navbar. Kindly address that. The TNW logo also would not be displayed in the medium screen size.

image

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes requested :recycle: 

#### Good job on the changes requested, and the page looks awesome :clap: You are almost there. Just fix a couple of problems that came from the new changes so the project looks even better (check the inline comments below).

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by nrqrmz** [comment]
Fix navbar and logo

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :green_circle: 

#### Great work on implemented the required changes :clap: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**